### PR TITLE
LDAP: Use InteractiveTable and remove gf-form usage

### PR DESCRIPTION
**Pull request by tskarhed**
**What is this feature?**

Deals with the LDAP related files in #65513.

Replaces the use of the HTML tables with grafana/ui's InteractiveTable
*Before*
image

*After*
image

**Why do we need this feature?**
Product consistency and  addressing technical debt. Some of the issues found here with InteractiveTable can be addressed in #80980

**Who is this feature for?**

LDAP people :)

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/admin/ldap/LdapUserMappingInfo.tsx [hunk 1]
```diff
@@ -8,40 +11,53 @@ interface Props {
...
+      id: 'cfgAttrValue',
+      header: 'LDAP attribute',
+    });
+  }
+
+  const rows = [
+    {
+      userInfo: 'First name',
+      ldapValue: info.name.ldapValue,
+      cfgAttrValue: info.name.cfgAttrValue,
```
[the remark this answers is not in the document]
Comment (tskarhed) [reply public/app/features/admin/ldap/LdapUserMappingInfo.tsx:35]: Also, is it ever false? 🤔

File: public/app/features/admin/ldap/LdapSyncInfo.tsx [hunk 2]
```diff
@@ -28,36 +28,33 @@ export class LdapSyncInfo extends PureComponent<Props, State> {
     const { isSyncing } = this.state;
```
Comment (tskarhed) [inline public/app/features/admin/ldap/LdapSyncInfo.tsx:28]: This syncing doesn't seem to do anything except altering UI state. Can it be removed? Should I also refactor this into a function component?

**Review by Jguer — APPROVED** [review]
Thanks @tskarhed

File: public/app/features/admin/ldap/LdapUserMappingInfo.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jguer) [reply public/app/features/admin/ldap/LdapUserMappingInfo.tsx]: I think it's always true indeed, can also be removed

**Review by Jguer — APPROVED** [review]

**Pull request merged**